tools/run_unit_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright 2021-present Extreme Networks, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Run stratum unit tests inside the stratum dev container.
+# Run from /stratum.
+
+TEST_CMD="xargs -a .circleci/test-targets.txt bazel test"
+# --flaky_test_attempts to retry transient admin_service_test failures
+TEST_CMD="$TEST_CMD --flaky_test_attempts 5"
+eval $TEST_CMD
```

Comment:
```
> shellcheck tools/run_unit_tests.sh

In tools/run_unit_tests.sh line 11:
eval $TEST_CMD
     ^-------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
eval "$TEST_CMD"
```

Reply:
Actually, we do not want TEST_CMD in quotes because we want the words split and globbing to happen. (shellcheck can be a little overreaching.) All your other comments make sense and I will reevaluate my approach.

Reply:
I see. You can leave this as is then, we'll re-check this once we add shellcheck to CI.